libs/wire-subsystems/src/Wire/MiniBackend.hs

```diff
@@ -52,28 +75,41 @@ newtype NotPendingStoredUser = NotPendingStoredUser StoredUser
 
 instance Arbitrary NotPendingStoredUser where
   arbitrary = do
-    user <- arbitrary
+    user <- arbitrary `suchThat` \user -> isJust user.identity
```

Comment:
i disagree.  we should do these things if we realize that the tests are too slow.

the reason why it's an antipattern is not coverage (we still generate the same number of *valid* values as before), but the fact that if you have very few random values satisfying your `suchThat`, your tests may not terminate (in time).  we don't have this problem here, the condition should be satisfied in way more than 50%.

Reply:
because it's not trivial (nothing is, not even this conversation) and serves no practical purpose?  sorry :).  let's discuss in person if *that* doesn't convince you.